app/scripts/TiledPlot.js

```diff
@@ -495,19 +495,18 @@ class TiledPlot extends React.Component {
     });
   }
 
-  handleOverlayMouseLeave() {
-    this.setState({
-      mouseOverOverlayUid: null,
-    });
+  handleOverlayMouseLeave(uid) {
+    if (uid === this.state.mouseOverOverlayUid) {
```

Comment:
Why is this needed? Shouldn't it not matter which track is left?

Reply:
It's necessary because of the order of events. Sometimes the dragStart event begins on an element before mouseLeave is called on the leaving element and that prevents the overlay from appearing. Took a surprisingly long time to figure that out actually.